src/main/interface/smartaudio_protocol.c

```diff
@@ -0,0 +1,194 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "platform.h"
+#include "common/crc.h"
+#include "interface/smartaudio_protocol.h"
+
+#define SMARTAUDIO_START_CODE               0xAA55
```

Comment:
SMARTAUDIO_START_CODE1 + SMARTAUDIO_START_CODE2 may be better (and will implicitly handle endianness)